src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -174,11 +181,20 @@ public ChassisSpeeds getRobotRelativeSpeeds(){
     }
     
     public void driveRobotRelative(ChassisSpeeds robotRelativeSpeeds){
-        var states = Constants.Swerve.swerveKinematics.toSwerveModuleStates(robotRelativeSpeeds);
-    
-        SwerveDriveKinematics.desaturateWheelSpeeds(states, Constants.Swerve.maxSpeed);
-    
-        setModuleStates(states);
+        //robotRelativeSpeeds.omegaRadiansPerSecond = 0.0;
+        SmartDashboard.putNumber("omega radians per second", robotRelativeSpeeds.omegaRadiansPerSecond);
+        SmartDashboard.putNumber("x speed", robotRelativeSpeeds.vxMetersPerSecond);
+        SmartDashboard.putNumber("y speed", robotRelativeSpeeds.vyMetersPerSecond);
```

Comment:
Let's go ahead and comment out these dashboard outputs.

Reply:
done